.env

```diff
@@ -25,4 +25,4 @@ SUPPORT_URL=''
 USER_INFO_COOKIE_NAME=''
 ENABLE_PROGRESS_GRAPH_SETTINGS=false
 ENABLE_TEAM_TYPE_SETTING=false
-ENABLE_NEW_EDITOR_PAGES=false
+ENABLE_NEW_EDITOR_PAGES=true
```

Comment:
This will turn on the urls (only reachable by staff) at https://stage.course-authoring.edx.org/course/<course_key>/editor/<blocktype>/<blockid>. These are 1. Only visible to staff 2. unreachable until we turn them on with waffle flags.